django/db/migrations/writer.py

```diff
@@ -472,6 +490,8 @@ def serialize(cls, value):
                     "For more information, see "
                     "https://docs.djangoproject.com/en/%s/topics/migrations/#serializing-values"
                     % (value.__name__, module_name, get_docs_version()))
+            if module_name == '__builtin__':
```

Comment:
Yes, you are correct.  I will open a separate ticket and pull request for this.

Do you have any suggestions on what function I could use in the test case?  The first one that comes to mind is `django.utils.crypto.get_random_string`.